packages/Box/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tds/core-box",
-  "version": "0.0.0-development",
+  "version": "1.0.0",
```

Comment:
Ohhhhh, I guess we misunderstood each other. We shouldn't bump the ACTUAL versions of the packages to 1.0 manually. In fact, we should NEVER change the "version" field in our packages manually. That must always be done by running `yarn version`. 

All I meant with my previous comment was to change the JSDoc 
"@version" to 1.0.0. But the packages themselves should stay at 0.0.0-development and we will bump them to 1.0.0 with the release/publish process.

Reply:
Makes sense. I just reverted the commit and added a separate commit to handle the v1.0.0 update on documentation only.